Firebase Push Notifications.md

```diff
@@ -0,0 +1,199 @@
+# Firebase push notifications
+
+We used to send push notifications directly to APNS or GCM from our code. We were maintaining device_ids, OS types, GCM/APNS certificates and sometimes this was really tricky.
+
+We knew that an easier solution exist and then we discovered the Firebase. Firebase is a service which does all hard work for us. There is no certificates at server side and we don't need to maintain OS types, because the Firebase knows which notification goes to the APNS and which to the GCM. Basically, Firebase handles the routing and delivery of push notifications to targeted devices. Firebase also has a dashboard with the statistics and mobile developers can use this dashboard for testing purposes.
```

Comment:
... discovered ~~the~~ Firebase.  `novi red`
Firebase is a service which does all `the` hard work for us. 
There `is no need to store certificates on the server side because Firebase handles them for us.`